src/openklant/components/klantinteracties/api/tests/test_filters.py

```diff
@@ -235,7 +236,7 @@ def test_filter_partij_partij_identificator_code_register(self):
         with self.subTest("happy flow"):
             response = self.client.get(
                 self.url,
-                {"hadBetrokkene__wasPartij__partijIdentificator__code_register": "hr"},
+                {"hadBetrokkene__wasPartij__partijIdentificator__codeRegister": "hr"},
```

Comment:
I guess this test was incorrectly succeeding previously, because the filter wasn't specified correctly? Could you check why that is and modify it to make sure it properly tests this filter?